src/python/pants/notes/master.rst

```diff
@@ -4,6 +4,22 @@ Master Pre-Releases
 This document describes ``dev`` releases which occur weekly from master, and which do
 not undergo the vetting associated with ``stable`` releases.
 
+1.10.0rc1 (09/17/2018)
```

Comment:
rcs past `0` don't occur from master, so they only edit the notes for the stable branch. Can drop all edits to this file.